dnf5/commands/download/download.hpp

```diff
@@ -47,6 +47,7 @@ class DownloadCommand : public Command {
     libdnf5::OptionBool * resolve_option{nullptr};
     libdnf5::OptionBool * alldeps_option{nullptr};
     libdnf5::OptionBool * url_option{nullptr};
+    libdnf5::OptionBool * urlallmirrors_option{nullptr};
```

Comment:
Other option variables are named after the options. Please rename `urlallmirrors_option' to `allmirrors_option'